Kickstarter-iOS/Views/Controllers/SheetOverlayViewController.swift

```diff
@@ -48,6 +52,12 @@ final class SheetOverlayViewController: UIViewController {
       |> \.backgroundColor .~ UIColor.ksr_support_700.withAlphaComponent(0.8)
   }
 
+  override func touchesBegan(_ touches: Set<UITouch>, with _: UIEvent?) {
```

Comment:
This function was added in here to enable tapping to dismiss. I looked at the other places we use this VC and didn't see any issues.